cfme/tests/infrastructure/test_host.py

```diff
@@ -312,3 +315,30 @@ def _finalize():
             actual_state, power_state
         ),
     )
+
+
+@test_requirements.infra_hosts
+@pytest.mark.meta(coverage=[1738664])
+@pytest.mark.parametrize(
+    "report_format", ["Download as Text", "Download as CSV", "Print or export as PDF"]
```

Comment:
**Required** I would add some ids so that the test case names aren't so long when they are generated from the parametrization. 

e.g. 
```python,
@pytest.mark.parametrize(
    "report_format", ["Download as Text", "Download as CSV", "Print or export as PDF"],
    ids=["txt", "csv", "pdf"]
)
```

Reply:
added ids.